schema/cassandra/cadence/schema.cql

```diff
@@ -475,3 +475,14 @@ CREATE TABLE queue (
 ) WITH COMPACTION = {
     'class': 'org.apache.cassandra.db.compaction.LeveledCompactionStrategy'
   };
+
+CREATE TABLE cluster_membership
+(
```

Comment:
partition key is missing

Reply:
I have been deferring partitionKey until after this and the cassandra frozen blob proto rework as I need to dig a bit deeper into the required changes on our side for partitioning and if there is anything additional for supporting vitess. 

When implementing partition keys for mysql is it correct that we would want to avoid spanning multiple partitions in queries? Assuming, so we'd want to pin all records to the same partition similarly to Cassandra, however, does the absence of a partition key not give the same behavior?

Reply:
Yes, we want all the records to live in a single partition. In case of Cassandra absence of partition key is anti-pattern: https://dba.stackexchange.com/questions/157537/querying-cassandra-without-a-partition-key

Reply:
Ah yes, had good convo with Samar on this, thanks you. I meant to reply to one of the .sql files in reference to postgres/mysql behavior without a PK.